assets/js/modules/analytics-4/components/settings/SettingsEnhancedMeasurementSwitch.test.js

```diff
@@ -111,7 +111,7 @@ describe( 'SettingsEnhancedMeasurementSwitch', () => {
 			] );
 	} );
 
-	it( 'should render with the switch defaulting to the on position when enhanced measurement is enabled for the web data stream', async () => {
+	it( 'should render with the switch defaulting to the on position when enhanced measurement is disabled for the web data stream', async () => {
```

Comment:
I don't think this change to the description needs to be made, as the test is still exercising the case where enhanced measurement is enabled for the stream, due to the call to `setEnhancedMeasurementStreamEnabled()` at the top of the test.

```suggestion
	it( 'should render with the switch defaulting to the on position when enhanced measurement is enabled for the web data stream', async () => {
```

Reply:
Good catch 👍